configuration/configuration.go

```diff
@@ -133,7 +139,23 @@ func LoadConfiguration(baseDirectory string) (*Configuration, error) {
 		Depth:     pruneDepth,
 		MinHeight: minPruneHeight,
 	}
-
+	defaultMaxSync := false
+	maxSyncValue := os.Getenv(MaxSyncConcurrency)
+	if len(maxSyncValue) == 0 {
+		defaultMaxSync = true
+	}
+	parsedValue, err := strconv.ParseInt(maxSyncValue, 10, 64)
+	if err != nil && !defaultMaxSync{
```

Comment:
The expression on line 148 is equivalent to:

!(err == nil || defaultMaxSync)  

if we want to change.